src/sql/workbench/common/styler.ts

```diff
@@ -49,3 +50,22 @@ export function attachTabbedPanelStyler(widget: IThemable, themeService: IThemeS
 		activeTabContrastBorder: cr.activeContrastBorder
 	}, widget);
 }
+
+export function attachCalloutDialogStyler(widget: IThemable, themeService: IThemeService, style?:
+	{
```

Comment:
Please make a type for this similar to https://github.com/microsoft/azuredatastudio/blob/main/src/vs/platform/theme/common/styler.ts#L52